NSpecifications.Tests/NSpecifications.Tests.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Nspecifications\NSpecifications.csproj" />
+    <ProjectReference Include="..\NSpecifications\NSpecifications.csproj" />
```

Comment:
The project wouldn't build for me without this change... It's not wholly necessary. I could have renamed the root repository folder.